bookkeeper-server/src/test/java/org/apache/bookkeeper/meta/TestLedgerMetadataSerDe.java

```diff
@@ -130,6 +130,7 @@ public void testPeggedToV2SerDe() throws Exception {
             .newEnsembleEntry(0L, Lists.newArrayList(new BookieSocketAddress("192.0.2.1", 3181),
                                                      new BookieSocketAddress("192.0.2.2", 3181),
                                                      new BookieSocketAddress("192.0.2.3", 3181)))
+            .withMetadataFormatVersion(LedgerMetadataSerDe.METADATA_FORMAT_VERSION_2)
```

Comment:
This part seems related

Cc @sijie

Reply:
You're correct that it's unrelated. Try to fix a test regression introduced by https://github.com/apache/bookkeeper/commit/407cb35e594e2334ae687f7c884651c7642349fa. It has been fixed separately. I'll revert this test change.